src/app/shell/header/header.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="container" fxLayout="column">
   <p-toolbar fxFill>
-    <div class="ui-toolbar-group-left" appFade="gt-xs" showDelay="0.3s" hideDuration="0.1s">
+    <div class="ui-toolbar-group-left" fxHide fxShow.xs appFade="gt-xs" showDelay="0.3s" hideDuration="0.1s">
```

Comment:
![image](https://user-images.githubusercontent.com/2085756/80892644-00a12800-8ccc-11ea-8d12-b466c440cca4.png)
tak łatwo sie nie poddam - będzie poprawiona dyrektywa, zgłoszony bug -> https://github.com/Wydatex/wydatex_frontend/issues/13
Z resztą tak jak to jest teraz też bez sensu bo dwie dyrektywy obserwują niepotrzebnie mediaQuery, jak już to całe appFade z parametrami powinno zostać wyrzucone (ale nie zostanie bo będzie naprawione! xd)
analogicznie pozostałe wrzutki fxHide / fxShow

Reply:
spoko, chciałem to teraz załatać ale jak to będziesz ogarniać to wywalę